vllm/plugins/__init__.py

```diff
@@ -25,6 +26,8 @@ def load_general_plugins():
     os.environ['TORCHINDUCTOR_COMPILE_THREADS'] = '1'
     # see https://github.com/vllm-project/vllm/issues/10619
     torch._inductor.config.compile_threads = 1
+    if current_platform.is_xpu():
+        os.environ['TORCH_COMPILE_DISABLE'] = 'True'
```

Comment:
```suggestion
        # see https://github.com/pytorch/pytorch/blob/8cada5cbe5450e17c26fb8b358116785324537b2/torch/_dynamo/config.py#L158 
        os.environ['TORCH_COMPILE_DISABLE'] = 'True'
```

Reply:
is it really working? `os.environ.get("TORCH_COMPILE_DISABLE", False)` will return `True` anyway. It returns a string.

Reply:
okay, as long as you set this env var, it will be disabled.